languages/common-lisp/concepts/named-parameters/about.md

```diff
@@ -1 +1,16 @@
-TODO: add information on named-parameters concept
+In Common Lisp named parameters are called keyword parameters.
+
+Keyword parameters are designated by the `&key` lambda list keyword in a lambda list. Keyword parameters are not required, can have a default and also can specify a "supplied-p parameter" which will be "true" or "false" depending on whether an argument was provided for the parameter.
```

Comment:
Have we explained lambda-lists somewhere? That's probably pretty foreign-sounding to non-lispers. I might be good to have in the functions concept (if we don't have it already).

Reply:
I defined 'lambda list' in the introduction to this exercise. But I can add something to the function concept as well.